make build-kfctl-container errors with cannot find module providing package

[Latest master ](https://github.com/kubeflow/kubeflow/commit/7c0932a117f2f2dc232271992d0348c4747021eb)doesn't build when running `make build-kfctl-container` from the bootstrap dir. I get:

```
#19 23.73 go: extracting github.com/kubeflow/kubeflow v0.6.1
#19 24.99 build github.com/kubeflow/kubeflow/bootstrap/v3/cmd/kfctlClient: cannot load github.com/kubeflow/kubeflow/bootstrap/cmd/bootstrap/app: cannot find module providing package github.com/kubeflow/kubeflow/bootstrap/cmd/bootstrap/app
#19 25.00 make: *** [Makefile:61: fmt] Error 1
------
executor failed running [/bin/sh -c make build-kfctl]: exit code: 2
make[1]: *** [Makefile:115: build-kfctl-container] Error 
```

This was working for me a month or so ago.

I can reproduce on master.

```
build github.com/kubeflow/kubeflow/bootstrap/v3/cmd/kfctlClient: cannot load github.com/kubeflow/kubeflow/bootstrap/cmd/bootstrap/app: cannot find module providing package github.com/kubeflow/kubeflow/bootstrap/cmd/bootstrap/app
make: *** [Makefile:61: fmt] Error 1
```

Test grid isn't showing any results (see kubeflow/testing#438)

Here are some postsubmit jobs though
https://prow.k8s.io/?repo=kubeflow%2Fkubeflow&type=postsubmit

--- a/bootstrap/Makefile
+++ b/bootstrap/Makefile
@@ -93,6 +93,9 @@ build-kfctl-tgz: build-kfctl
 	cd bin && tar -cvzf kfctl_$(TAG)_linux.tar.gz ./kfctl
 	cp -f ./bin/kfctl_$(TAG)_linux.tar.gz ./bin/kfctl_$(TAG)_darwin.tar.gz
 
+build-kfctl-client: /tmp/v2 deepcopy generate fmt vet
+	${GO} build -i -gcflags 'all=-N -l' -ldflags "-X main.VERSION=$(TAG)" -o bin/kfctlClient cmd/kfctlClient/main.go
+
 # push the releases to a GitHub page
 push-to-github-release: build-kfctl-tgz
 	github-release upload \

--- a/bootstrap/cmd/kfctlClient/README.md
+++ b/bootstrap/cmd/kfctlClient/README.md
@@ -1,0 +1,7 @@
+# kfctl client
+
+A simple go binary to send requests to the kfctl server.
+
+This is primarily intended for development and testing and not for actual user.
+
+We should consider adding an appropriate subcommand to kfctl

--- a/bootstrap/cmd/kfctlClient/main.go
+++ b/bootstrap/cmd/kfctlClient/main.go
@@ -1,0 +1,185 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	log "github.com/golang/glog"
+	"github.com/kubeflow/kubeflow/bootstrap/v3/cmd/bootstrap/app"
+	kfdefsv2 "github.com/kubeflow/kubeflow/bootstrap/v3/pkg/apis/apps/kfdef/v1alpha1"
+	"github.com/kubeflow/kubeflow/bootstrap/v3/pkg/kfapp/gcp"
+	"github.com/kubeflow/kubeflow/bootstrap/v3/pkg/utils"
+	"github.com/pkg/errors"
+	"golang.org/x/oauth2"
+	"os"
+
+	// log "github.com/sirupsen/logrus"
+	"golang.org/x/oauth2/google"
+	dm "google.golang.org/api/deploymentmanager/v2"
+)
+
+// ServerOption is the main context object for the controller manager.
+type ServerOption struct {
+	Project  string
+	Name     string
+	Config   string
+	Endpoint string
+	Zone     string
+}
+
+// NewServerOption creates a new CMServer with a default config.
+func NewServerOption() *ServerOption {
+	s := ServerOption{}
+	return &s
+}
+
+// AddFlags adds flags for a specific Server to the specified FlagSet
+func (s *ServerOption) AddFlags(fs *flag.FlagSet) {
+
+	fs.StringVar(&s.Config, "config", "https://raw.githubusercontent.com/kubeflow/kubeflow/master/bootstrap/config/kfctl_gcp_iap.yaml", "URI of a YAML file containing a KfDef object.")
+	fs.StringVar(&s.Name, "name", "", "Name for the deployment.")
+	fs.StringVar(&s.Project, "project", "", "Project.")
+	fs.StringVar(&s.Endpoint, "endpoint", "", "The endpoint e.g. http://localhost:8080.")
+	fs.StringVar(&s.Zone, "zone", "", "Zone.")
+
+}
+
+func checkAccess(project string, token string) {
+	// Verify that user has access. We shouldn't do any processing until verifying access.
+	ts := oauth2.StaticTokenSource(&oauth2.Token{
+		AccessToken: token,
+	})
+
+	isValid, err := app.CheckProjectAccess(project, ts)
+
+	if err != nil || !isValid {
+		log.Fatalf("CheckProjectAccess failed; error %v", err)
+	}
+
+	log.Infof("You have access to project %v", project)
+}
+func run(opt *ServerOption) error {
+	if opt.Name == "" {
+		return fmt.Errorf("--name is required.")
+	}
+
+	if opt.Project == "" {
+		return fmt.Errorf("--project is required.")
+	}
+
+	d, err := kfdefsv2.LoadKFDefFromURI(opt.Config)
+
+	if err != nil {
+		return errors.WithStack(err)
+	}
+
+	d.Spec.Project = opt.Project
+	d.Name = opt.Name
+
+	email, err := gcp.GetGcloudDefaultAccount()
+
+	if err != nil {
+		log.Errorf("Couldn't get default email; error %v", err)
+	}
+
+	log.Infof("Setting email to %v", email)
+	d.Spec.Email = email
+
+	fmt.Printf("Connecting to server: %v", opt.Endpoint)
+	c, err := app.NewKfctlClient(opt.Endpoint)
+
+	if err != nil {
+		log.Errorf("There was a problem connecting to the server %+v", err)
+		return err
+	}
+
+	if os.Getenv(gcp.CLIENT_ID) == "" {
+		log.Errorf("Environment variable CLIENT_ID must be set for IAP")
+		return fmt.Errorf("Must set environment variable CLIENT_ID for IAP")
+	}
+
+	if os.Getenv(gcp.CLIENT_SECRET) == "" {
+		log.Errorf("Environment variable CLIENT_SECRET must be set for IAP")
+		return fmt.Errorf("Must set environment variable CLIENT_SECRET for IAP")
+	}
+
+	// Set the GCPPluginSpec.
+	pluginSpec := &gcp.GcpPluginSpec{}
+	if err := d.GetPluginSpec(gcp.GcpPluginName, pluginSpec); err != nil && !kfdefsv2.IsPluginNotFound(err) {
+		return err
+	}
+
+	pluginSpec.Auth = &gcp.Auth{
+		IAP: &gcp.IAP{
+			OAuthClientId: os.Getenv(gcp.CLIENT_ID),
+			OAuthClientSecret: &kfdefsv2.SecretRef{
+				Name: gcp.KUBEFLOW_OAUTH,
+			},
+		},
+	}
+
+	if err := d.SetPluginSpec(gcp.GcpPluginName, pluginSpec); err != nil {
+		return err
+	}
+
+	d.SetSecret(kfdefsv2.Secret{
+		Name: gcp.KUBEFLOW_OAUTH,
+		SecretSource: &kfdefsv2.SecretSource{
+			LiteralSource: &kfdefsv2.LiteralSource{
+				Value: os.Getenv(gcp.CLIENT_SECRET),
+			},
+		},
+	})
+
+	ts, err := google.DefaultTokenSource(context.Background(), dm.CloudPlatformScope)
+
+	if err != nil {
+		return err
+	}
+
+	token, err := ts.Token()
+
+	if err != nil {
+		return err
+	}
+
+	d.SetSecret(kfdefsv2.Secret{
+		Name: gcp.GcpAccessTokenName,
+		SecretSource: &kfdefsv2.SecretSource{
+			LiteralSource: &kfdefsv2.LiteralSource{
+				Value: token.AccessToken,
+			},
+		},
+	})
+
+	d.Spec.Zone = opt.Zone
+
+	fmt.Printf("Spec to create:\n%v", utils.PrettyPrint(d))
+
+	checkAccess(opt.Project, token.AccessToken)
+
+	// TODO(jlewi) continually retry and wait for success or failure
+	ctx := context.Background()
+	res, err := c.CreateDeployment(ctx, *d)
+
+	if err != nil {
+		log.Errorf("CreateDeployment failed; error %v", err)
+		return err
+	}
+
+	log.Infof("Create succedeed. Result:\n%v", utils.PrettyPrint(res))
+	return nil
+}
+
+func main() {
+	s := NewServerOption()
+	s.AddFlags(flag.CommandLine)
+
+	flag.Parse()
+
+	err := run(s)
+
+	if err != nil {
+		log.Errorf("Create deployment failed; error %v", err)
+	}
+}

--- a/bootstrap/go.mod
+++ b/bootstrap/go.mod
@@ -13,9 +13,11 @@ require (
 	github.com/go-openapi/spec v0.19.2 // indirect
 	github.com/go-openapi/swag v0.19.2 // indirect
 	github.com/gogo/protobuf v1.2.1
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/golang/protobuf v1.3.1
 	github.com/hashicorp/go-getter v1.0.2
 	github.com/imdario/mergo v0.3.7
+	github.com/kr/pty v1.1.3 // indirect
 	github.com/kubeflow/kubeflow/components/profile-controller v0.0.0-20190614045418-7ca3cfb39368
 	github.com/mailru/easyjson v0.0.0-20190312143242-1de009706dbe // indirect
 	github.com/mitchellh/go-homedir v1.1.0

